<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyTrader v0.2</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>181</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>수동주문</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>계좌</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>15</y>
       <width>120</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>75</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>주문</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>70</y>
       <width>120</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>신규매수</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>신규매도</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>매수취소</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>매도취소</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>105</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>종목</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>100</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>160</y>
       <width>120</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>지정가</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>시장가</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>165</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>종류</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>195</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>수량</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>190</y>
       <width>120</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>225</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>가격</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_2">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>220</y>
       <width>120</width>
       <height>22</height>
      </rect>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="maximum">
      <number>10000000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>255</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>현금주문</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>130</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 255);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>47</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>PW</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_PW">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>42</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_TEST">
     <property name="geometry">
      <rect>
       <x>92</x>
       <y>255</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>TEST</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>291</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>조건 검색</string>
    </property>
    <widget class="QComboBox" name="comboBox_cond">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_cond">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>261</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>로그창</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cond">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>적용</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_cond">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>텔레그램 알림받기</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>291</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>조건 검색2</string>
    </property>
    <widget class="QComboBox" name="comboBox_cond_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_cond_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>261</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>로그창</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cond_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>적용</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_cond_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>텔레그램 알림받기</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>340</y>
      <width>291</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>조건 검색3</string>
    </property>
    <widget class="QComboBox" name="comboBox_cond_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_cond_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>261</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>로그창</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cond_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>적용</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_cond_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>텔레그램 알림받기</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>340</y>
      <width>291</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>조건 검색4</string>
    </property>
    <widget class="QComboBox" name="comboBox_cond_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_cond_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>261</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>로그창</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cond_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>적용</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_cond_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>텔레그램 알림받기</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
